torchvision/transforms/functional.py

```diff
@@ -173,7 +175,7 @@ def to_tensor(pic) -> Tensor:
         return img
 
 
-def pil_to_tensor(pic: Any) -> Tensor:
+def pil_to_tensor(pic: Any, device: torch.device = "cpu") -> Tensor:
     """Convert a ``PIL Image`` to a tensor of the same type.
```

Comment:
The docstring below mentions the type of pic is only PIL.Image but over here it is mentioned Any.
Please consider changing this